### DISCUSSION TITLE
Milestone 1 事项讨论

### DISCUSSION THREAD

**Discussion by creamidea** [cellbang/malagu#34] [Ideas]
第一个里程碑，时间跨度：2021-07-01 ~ 2021-12-30

- 组建技术小组，确立负责人
   - 确定 feature/bug 提交规范和跟进制度
   - 确定 PR 提交规范和审阅规范
   - 建立日常迭代机制
   - 完善开发者文档

- 组建影响力洞察小组，确立负责人
   - 客户联络和回访机制，收集诉求（现有的客户可以整理一下）
   - 可以做出一个具有影响力的方案并落地（比如联系一些行业的头部企业，达成合作协议）
   - 寻找相关大会分享机会，公众号、知乎等等舆论阵地的建设

PS：上述只是我个人给出的方向和参考，负责人可以根据情况增删。

**Comment by creamidea** [comment]
技术组
成员 @muxiangqiu  跃羽 吴林胜 @2234839

运作（影响力&用户）组
成员 业 Jaryn @creamidea

**Reply by nexa-li** [reply]
我同意加入运作（影响力&用户）组

**Reply by creamidea** [reply]
我近期可以负责运作（影响力&用户）方向，主要的思路：

- 协助落地 @JarynChen DDD 探索（后续这个可以作为一次大会分享）
- 项目 blog 体系、公众号和知乎专栏建设 @业

**Comment by creamidea** [comment]
首先，麻烦 @muxiangqiu 创建 2 个小组，技术组和运作组。然后把我们加入进去

技术组的职责：feature 开发，bug 修复，技术问题回答，技术文档编写和维护等
运作组的职责：管理和跟踪目前使用用户的问题和诉求（技术组和用户之间的桥梁），维持日常群里的秩序和活跃度，建立宣传渠道等

![image](https://user-images.githubusercontent.com/1274730/124386124-655e7500-dd0b-11eb-8e8c-bb2db1d7a5f8.png)

根据群里用户的反馈，以及之前 malagu 的运行情况，这一阶段的重点是 **用户&文档体系** ，优化目前的文档体系和用户入门体感。(目前看，功能开发已经进入日常迭代阶段，Web 领域开发需要的功能基本具备）

大致的路线
![image](https://user-images.githubusercontent.com/1274730/124386526-82944300-dd0d-11eb-8ce2-a6271cf36499.png)

在 0711 这个节点前，需要完成（建议 0709 的晚上可以预约组内所有同学视频评审一下，或者写出文档异步评审，看你们的时间）
- 用户画像（根据目前的用户分析，他们由哪几类人群组成） @业
- 文档体系（需要分哪些栏目，同时确定每个人需要完成的内容）@muxiangqiu （我看群里面你在跟进这个，所以先安排给你了，需要的是一个宏观层面的设计） 

节点 0725 是按照 0711 的安排完成的时间点，这一天应该要写的文档都会写完，变成草稿的状态

节点 0802 正式发布新的文档。预留一周的时间给大家 review

之后就是从各个渠道收集反馈意见，安排下一个迭代

借这次文档更新的契机，给 malagu 做一次宣发。另外，@JarynChen 可以继续聚焦在 DDD 的方向探索上，可以每个月跟群里其他同学同步一下进度和想法，当然也可以做一些宣发上面的事情。
